backend/src/docs/v1.api-spec.yaml

```diff
@@ -1,4 +1,4 @@
-openapi: 3.0.0
+openapi: 3.0.2
```

Comment:
Not a requirement but adding the YAML `---` demarker would be good for consistency.

Reply:
Added